File: leo2-node/src/main/java/org/deku/leo2/node/data/repositories/master/RoutingLayerRepository.java
package org.deku.leo2.node.data.repositories.master;

import org.deku.leo2.node.data.entities.master.RoutingLayer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

/**
 * Created by JT on 23.06.15.
 */
public interface RoutingLayerRepository extends JpaRepository<RoutingLayer,Integer>,QueryDslPredicateExecutor
{
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repository/RoutingLayerRepository.kt
package org.deku.leoz.node.data.repository

import org.deku.leoz.node.data.jpa.MstRoutingLayer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

/**
 * Routing layer repository
 * Created by JT on 23.06.15.
 */
interface RoutingLayerRepository :
        JpaRepository<MstRoutingLayer, Int>,
        QuerydslPredicateExecutor<MstRoutingLayer>


